.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.md 
+!README.md
```

Comment:
Great job making the changes requested by the previous reviewer but there is a slight correction to do 😦 .

As stated by the previous reviewer, kindly add the 

> test.md to gitignore

I noticed you try adding `.md `. You can add it this way `test.md` 😄 

Also, only two items should be added to gitignore and these are 
`node_modules` and `test.md`. Kindly remove the `README.md` from gitignore 😊